lib/Pool.js

```diff
@@ -171,6 +171,8 @@ Pool.prototype.end = function (cb) {
 Pool.prototype.query = function (sql, values, cb) {
   var query = Connection.createQuery(sql, values, cb);
 
+  query.isFormatted = false;
```

Comment:
Move this so it always exists, otherwise we are altering the hidden class and fracturing it.